modules/static/google/gnostic/rsync.incl

```diff
@@ -0,0 +1,11 @@
++ /LICENSE
+
+# Open API v2
++ /openapiv2/OpenAPIv2.proto
+
+# Open API v3
++ /openapiv3/annotations.proto
```

Comment:
Present since release v0.6.8: https://github.com/bufbuild/modules/pull/550#discussion_r1600306322

Reply:
<img width="255" alt="image" src="https://github.com/bufbuild/modules/assets/398111/de3fa9e0-8eda-4d50-9977-03cf3b14e26a">

Do we not want all of these .proto files?

Reply:
Also, I think we'd want to move them to proper folders based on the package names. For example, `/openapiv2/OpenAPIv2.proto` defines package `openapi.v2` so the file should be moved to `openapi/v2/OpenAPIv2.proto`, right?

Reply:
> Do we not want all of these .proto files?

I did not add them because they were not requested in the original issue #482, just the `openapiv2` and `openapiv3` directories. cc @bufdev can you confirm if we also want those files pls?

> I think we'd want to move them to proper folders based on the package names. For example, `/openapiv2/OpenAPIv2.proto` defines package `openapi.v2` so the file should be moved to `openapi/v2/OpenAPIv2.proto`

That would be a first, we usually respect the original directory structures for all other managed modules from source (which they usually match with the pkg name) I can move it to the correct directory, sure, but I think that would result in a mismatch for users looking to depend on them, import them, and realize the paths don't match with source, wdyt?

Reply:
[This](https://buf.build/gnostic/gnostic/docs/main:gnostic.openapi.v3) is the code that was requested for syncing, however, unless the module is untenably large I think we should default to including all relevant proto definitions.

Reply:
Added the relevant protos (excluding `examples` and `third_party` dirs), test-synced them, this is the most recent synced ref `v0.7.0`: https://github.com/bufbuild/modules/pull/551#discussion_r1600417186

And about the movement of file to match their proto pkgs, most of those other synced protos have pkg names that also don't match their parent dirs:

| File path | Proto pkg |
|--------|--------|
| discovery/discovery.proto | package discovery.v1; |
| extensions/extension.proto | package gnostic.extension.v1; |
| metrics/lint/linter.proto | package linter; |
| metrics/complexity.proto | package gnostic.metrics.v1; |
| plugins/plugin.proto | package gnostic.plugin.v1; | 
| surface/surface.proto | package surface.v1; | 

Moving all those files to their respective "expected dirs" according to the pkg names would result in a big change vs the original source paths. I'd say we should respect the dir structure from source?

Reply:
The files in the original repo are in the proper locations: https://buf.build/gnostic/gnostic/tree/main:gnostic/openapi/v3

I think we should match if this is intended to replace that repo.